src/Fabulous.AST/Widgets/TypeDefinitions/Class.fs

```diff
@@ -54,7 +54,7 @@ module Class =
                             None,
                             None,
                             SingleTextNode("(", Range.Zero),
-                            simplePatNodes,
+                            List.map Choice1Of2 simplePatNodes,
```

Comment:
This will not work when there are multiple parameters.
[Example](https://fsprojects.github.io/fantomas-tools/#/oak?data=N4KABGBEDGD2AmBTSAuKAXAngB0WAYrLABQCGANAEYCUYAvGNADakDOrYiAdvJOeFCQAzAJZdErVFD4DII1vlYipQ0k1aJ%2BEOawDiAJ1LYAFgDURiAO4q1GkAF8gA)

It should be something along the lines of 
```fsharp
let simplePats =
    match simplePatNodes with
    | [] -> []
    | head :: tail -> 
          [ yield Choice1Of2 head
            for p in tail do
                yield Choice2Of2 (SingleTextNode(",", Range.Zero))
                yield Choice1Of2 p ]
```

Reply:
I have created a new issue for this. Thanks